hugo/assets/scss/config/colors.scss

```diff
@@ -2,7 +2,7 @@ $c-blue--lightest:      #f8f9fd;
 $c-blue--lighter:       #f0f2fb;
 $c-grey--light-mid:     #e6e8f3;
 $c-blue--light:         #1b3987;
```

Comment:
Now blue--light and blue are the same. Also the variable $c-grey--light-mid is wrongly named because it's also one of the blues. So let's clean this up a bit because otherwise it's confusing for the future:

$c-blue--light:         #1b3987;  is now a duplicate of $c-blue and can be removed.
$c-grey--light-mid > rename to $c-blue--light

So after that change we will have: 

```$c-blue--lightest:      #f8f9fd;
$c-blue--lighter:       #f0f2fb;
$c-blue--light:         #e6e8f3;
$c-blue:                #1b3987;
$c-blue--bright:        #063AC0;
$c-blue--dark:          #0c2c65;
$c-blue--darker:        #0f1333;
```

Reply:
Updated colors file and one affected scss file (trees).